source/CommandLine/CommandLocator.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Octopus.CommandLine.Commands;
+
+namespace Octopus.CommandLine
+{
+    public class CommandLocator : ICommandLocator
```

Comment:
This has changed. Worth doing a diff here.
We used to take in an `ILifetimeScope`, scan the assembly ourselves, then manually resolve from the container which was sucky. 
Now we take in `IEnumerable<ICommand>` instead.